src/zope/app/apidoc/tests.py

```diff
@@ -180,7 +180,9 @@ def test_slot_methods2(self):
     @unittest.skipIf(str is bytes, "Only on CPython3")
     def test_slot_methods3(self):
         from zope.app.apidoc.utilities import getFunctionSignature
-        self.assertEqual("(self, *args, **kwargs)", getFunctionSignature(object.__init__))
+        self.assertEqual(
+            '(self, /, *args, **kwargs)',
```

Comment:
It has to do with [positional only](https://www.python.org/dev/peps/pep-0457/) arguments...which I know doesn't really make much sense for `object.__init__`, but neither did the `(self, *args, **kwargs)` variant since we know it doesn't actually take varargs. 

There was a recent discussion of how Signature still gets this wrong on PythonDev for some C implementations (apparently argument clinic does better, IIRC?) but how it's still needed internally...or something; I don't exactly remember, and it's proving very resistant to finding by my google-fu.